src/wallet/walletdb.cpp

```diff
@@ -1011,9 +1013,11 @@ std::unique_ptr<WalletDatabase> MakeDatabase(const fs::path& path, const Databas
 
     Optional<DatabaseFormat> format;
     if (exists) {
+#ifdef USE_BDB
         if (ExistsBerkeleyDatabase(path)) {
             format = DatabaseFormat::BERKELEY;
         }
+#endif
 #ifdef USE_SQLITE
```

Comment:
No, it's supposed to `#ifdef ... #ifdef` because both `USE_BDB` and `USE_SQLITE` can both be set, and in fact, is the default.

The instances where `#ifdef...#elif` is used is only because one wallet type must be defaulted to, and for now, we default to BDB.